thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -220,6 +220,15 @@ public void run() {
       alertConfig = alertConfigDAO.findById(configuration.getLong(ALERT_ID));
       EmailConfig emailConfig = alertConfig.getEmailConfig();
       emailConfig.getFunctionIds().add(anomalyFunction.getId());
+
+      // Add recipients to existing alert group
+      String recipients = configuration.getString(ALERT_TO);
+      if (StringUtils.isNotBlank(recipients)) {
+        if (StringUtils.isNotBlank(alertConfig.getRecipients())) {
+          recipients = recipients + "," + alertConfig.getRecipients();
```

Comment:
Hi Yung-Yu, thanks for fixing this. do we check if the input recipients already exists in current recipient list?

Reply:
That's a good point. Currently, it is no harm to add duplicate emails. But to check if the email is redundant, should we also check if the email format is valid?

Reply:
It is done. Please review the change.